samples/sensor/adt7420/README.rst

```diff
@@ -8,7 +8,12 @@ Description
 
 This sample application periodically (1Hz) measures the ambient temperature
 in degrees Celsius. The result is written to the console.
-Optionally, it also shows how to use the upper and lower threshold triggers.
+
+When configured in trigger mode the update interval is 5 s, and the
+sample maintains a 1 Cel window around a recent temperature.  As soon as
```

Comment:
I'm not sure which doc changes are going to win since I see doc changes here and in #21558.